guides/source/form_helpers.md

```diff
@@ -108,7 +108,7 @@ This generates the following:
 <label for="pet_cat">I own a cat</label>
 ```
 
-The first parameter to [`check_box`](https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-i-check_box) is the name of the input. The second parameter is the value of the input. This value will be included in the form data (and be present in `params`) when the checkbox is checked.
+The first parameter to [`check_box`](https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-i-check_box) is the name of the input. The second parameter is a hash of options and the third parameter is the value of the checked input. This value will be included in the form data (and be present in `params`) when the checkbox is checked.
```

Comment:
Since the API documentation concisely describes the method signature, let's lean on that.  We can hint to the reader what is possible, and then suggest where to look for more information.

```suggestion
The first parameter to [`check_box`](https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-i-check_box) is the name of the input. The checkbox's values (the values that will appear in `params`) can optionally be specified using the third and fourth parameters. See the API documentation for details.
```

Reply:
I'm glad you suggested this because it felt a bit much to go through all the args as I was updating it. I like the idea of pointing the reader to the API documentation for more information.